guide/payments/privacy.md

```diff
@@ -0,0 +1,139 @@
+
+The key to keeping your transactions private is to prevent others from determining which addresses you own[^3]. Satoshi let others know they mined the first block, which contained just one transaction, with the block reward getting sent to their address. This illustrates the possible permanence of association between an address and an identity. That being said, the pseudonym “Satoshi Nakamoto” also contributes to further anonymity. While it's possible break asumptions of ownership going forward, the challenge is to recover privacy once an association is made public.
+
+<!-- TODO: add graphic and heading that demonstrate points of compromise when transacting with bitcoin -->
+
+## Methods to preserve privacy
+
+Keeping private takes constant and diligent work but is not impossible to ensure an acceptable level of privacy[^1]. Let’s explore some methods and practices that help preserve privacy on a transaction level.
+
+<!-- talk about the problem as you are talking about the solution -->
+
+### Generate a new address for each payment
+
+Creating new online personas is not the only way to preserve privacy. A new address can be generated by the wallet application used to receive bitcoin. This is achieved using [HD Wallets]({{ "/guide/glossary/#hd-wallet" | relative_url }}), a standard for how private keys get generated in modern bitcoin applications. This allows each transaction not to be tied to a single address, making it difficult to associate with to the owner.
+
+> If thieves can see your income, holdings, and spending, they can use this information to target and exploit you.[^2]
+
+Using the same address multiple times does not only degrade the privacy of just one transacting party. Both the [sender](/guide/payments/receive) and [receiver](/guide/payments/receive) are put at risk. On the receiver’s side, by using a single address, senders or anyone with whom the address is shared can see the previous activity and the amount of bitcoin controlled by the owner.
+
+Assuming an adversary uncovers the identity behind the reused address there is now a single point for them to reach anyone else who has transacted with the owner.
+
+<!--
+TODO: Link / mention gap limit
+TODO: Graphic / consider how to get the ui generating multiple addresses. make it easy for the user to not fail 
+
+> ###### Worked Example 1 - Savings Revealed
+> * You save in bitcoin, using a single-address paper wallet.
+> * All your bitcoin savings to this same address, let's say it contains $1 million worth.
+> * You buy a small amount of bitcoins to add to your savings, depositing in the paper wallet.
+> * The person who sold you the bitcoins follows their trail on the blockchain and finds your paper wallet containing $1 million.
+> * He mentions it to someone in a cafe or bar.
+> * Word gets around. A burglar raids your home. Kidnappers capture your children and know exactly how much to demand in ransom.
+
+-->
+
+
+**Do's**
```

Comment:
Should be more clear that these are design do's - the content before this doesn't add that context either.

Reply:
The whole guide is about design and we use these do's and don'ts pretty consistently.